lib/src/html/template_data.dart

```diff
@@ -32,7 +32,10 @@ abstract class TemplateData<T extends Documentable> {
   String get title;
   String get layoutTitle;
   String get metaDescription;
+
   List get navLinks;
+  Documentable get navLinksLast => navLinks.last;
```

Comment:
Renamed to `parent`, and added some more semantic accessors on `TemplateData` (`name` and `kind`).